checks.nix

```diff
@@ -38,20 +46,19 @@ let
       ) set
     );
 
-  isUnfreeRedistributable =
-    licenses:
-    lib.lists.any (
-      l:
-      let
-        free = l.free or true;
-        redistributable = l.redistributable or false;
-      in
-      !free && redistributable
-    ) (lib.lists.toList licenses);
+  isUnfree = pkg: lib.lists.any (l: !(l.free or true)) (lib.lists.toList (pkg.meta.license or [ ]));
+
+  isSource =
+    key: pkg: !lib.lists.any (x: !(x.isSource)) (lib.lists.toList (pkg.meta.sourceProvenance or [ ]));
+
+  isNotLinuxKernel = key: !(lib.hasPrefix "linuxKernel_" key);
```

Comment:
Looks like we'll need to filter both `linuxKernel_` and `linuxPackages`.

Reply:
ok, fixed!